source/text/labelrenderpass.ts

```diff
@@ -215,7 +219,8 @@ export class LabelRenderPass extends Initializable {
             gl.uniform1f(this._uAAStepScale, this._aaStepScale);
         }
 
-        let labelsAltered = override || this._altered.labels;
+        /** Some labels need the camera to update their font size and position */
```

Comment:
`/**` .. only one `/*` and inline code comments in webgl-operate always start with Capital letter and end with dot `.`